src/app/services/map-provider.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+
+export interface MapProvider {
+  url: string;
+  copyright: string;
+}
```

Comment:
doesn't necessarily need to change, do we can a convention around interfaces? Inconsistency here is `MapLocation` is in a model file while this is just in with a service

Reply:
I put it as a service, because this is (an interface for) an injectable object, and it made sense to keep it in the same file as the classes that implement it. On the other hand, MapLocation just stores the data.

Happy to change this, I'm open to suggestions.

Reply:
Works for me!